handbook/engineering/testing.md

```diff
@@ -0,0 +1,8 @@
+# Testing
+
+This file documents how we test code at Sourcegraph.
+
+
+## Percy
+
+We use [Percy](https://percy.io/) to detect visual changes in Sourcegraph features. You may need permissions to update screenshots if your feature introduces visual changes. Post a message in #dev-chat that you need access to Percy, and someone will add you to our organization (you will also receive an invitation via e-mail). Make sure you are logged in with your GitHub credentials.
```

Comment:
Hmm...so docs.sourcegraph.com is for product docs so developer related documentation doesn't really belong there (e.g. we should perhaps move it out of the docs directory, but you don't necessarily need to do this yourself).

I agree that the handbook should have *some* page that talks about testing, but it also feels wrong to include docs that are hyper specific to sourcegraph/sourcegraph (which is what https://sourcegraph.sgdev.org/sourcegraph/sourcegraph/-/blob/doc/dev/testing.md is).

How about we just link to it from here. I think that would resolve my concerns for now.

```suggestion
We use [Percy](https://percy.io/) to detect visual changes in Sourcegraph features. You may need permissions to update screenshots if your feature introduces visual changes. Post a message in #dev-chat that you need access to Percy, and someone will add you to our organization (you will also receive an invitation via e-mail). Make sure you are logged in with your GitHub credentials.

## Other

[Documentation for running tests in sourcegraph/sourcegraph](https://sourcegraph.sgdev.org/sourcegraph/sourcegraph/-/blob/doc/dev/testing.md).
```

Reply:
Works for me!